### ISSUE TITLE
Linked Libraries not showing up

### ISSUE THREAD

**Issue by tannav** [kawamurakazushi/figma-walker#31]
Would it be possible to show linked libraries when using Figma Walker (with the "i" setting)

**Comment by davidprati** [comment]
I agree, more and more companies are working with linked libraries :)
